### PR TITLE
[ETP-811] allow the count to update even if not public

### PR DESCRIPTION
**Pull request by ScottGarrison**
### 🎫 Ticket

[[ETP-811](https://theeventscalendar.atlassian.net/browse/ETP-811)] 

### 🗒️ Description

Fixes the issues where while using `tribe_tickets_attendees` shortcode, 0 attendees always shows up.
Did so by matching how the Block version was handling the variables, then matched the template variable being passed because it wasn't matching.

I also removed the public call to the Attendee List query as it only removes all the avatars from showing. The user being able to hide their avatar when purchasing a ticket still functions properly. 

### 🎥 Artifacts 
https://www.loom.com/share/bb1b0c481ea5411bb98a91c973cefc86

### ✔️ Checklist
- [✔️] I've included a changelog entry in the readme.txt file. 
- [✔️] My code is tested. 
- [✔️] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ETP-811]: https://theeventscalendar.atlassian.net/browse/ETP-811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
 == Changelog ==
 
+= [TBD]
+* Fix - Fixes the issues where while using `tribe_tickets_attendees` shortcode, 0 attendees always shows up.
```
Comment (codingmusician) [inline changelog.txt:4]:
```suggestion
* Fix - Fixes the issues where while using `tribe_tickets_attendees` shortcode, 0 attendees always shows up. [ETP-811]
```

**Review by bordoni — DISMISSED** [review]

**Comment by bordoni** [comment]
We need to make sure we point to the correct branch here.

**Comment by bordoni** [comment]
We need to regenerate the Snapshots for Commerce Integration.

**Comment by rafsuntaskin** [comment]
This one is resolved with this PR: https://github.com/the-events-calendar/event-tickets-plus/pull/1383

**Pull request closed without merging**